### DISCUSSION TITLE
server: streaming like Twitch

### DISCUSSION THREAD

**Discussion by Tau5** [asciinema/discussions#200] [Ideas]
We alredy have offline streaming, it would be a good a addition to be able to stream a session in a asciinema server (ej.: asciinema.org) and have a chat to viewers to talk.

You could, for example: stream a gameplay of NetHack, or stream how you develop an aplication.

**Comment by ku1ik** [comment]
That's definitely something I plan to implement 👍

**Comment by pratikbin** [comment]
Waiting for this one too

**Comment by ku1ik** [comment]
FYI, you can now live stream your terminal with asciinema CLI 3.0 (https://github.com/asciinema/asciinema/releases/tag/v3.0.0-rc.4). Check out `asciinema stream --help` or `asciinema session --help`.
